### PR TITLE
Allow specifying requiredPermission as string oder array of strings

### PR DESCRIPTION
**Pull request by fraxachun**
Depends on https://github.com/vivid-planet/comet/pull/1431

### CODE REVIEW

**Review by nsams — DISMISSED** [review]

File: packages/api/cms-api/src/generator/crud-generator.decorator.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export interface CrudGeneratorOptions {
     targetDirectory: string;
-    requiredPermission?: [string];
+    requiredPermission?: [string] | string;
```
[the remark this answers is not in the document]
Comment (johnnyomair) [reply packages/api/cms-api/src/generator/crud-generator.decorator.ts:3]: And yes, GraphQL uses `[String]`, which is very confusing, but necessary 😁

[the remark this answers is not in the document]
Comment (fraxachun) [reply packages/api/cms-api/src/generator/crud-generator.decorator.ts]: 0699fd2c7087f0f43ba96246c47efb3774499318

**Comment by johnnyomair** [comment]
@fraxachun do you need to rebase this PR? It still has commits from #1431.

**Review by nsams — DISMISSED** [review]

**Comment by fraxachun** [comment]
> @fraxachun do you need to rebase this PR? It still has commits from #1431.

Rebased.

**Review by nsams — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**